### PR TITLE
added methods to display data from person class

### PR DESCRIPTION
**Pull request by kblack89**
This code is creating two methods to take the data from the person class and display the information

### CODE REVIEW

File: heavenly-visions/Person.cs
Comment (Loerad) [inline heavenly-visions/Person.cs:1]: this code can be voided

**Review by Loerad — APPROVED** [review]
Some merge conflicts that can be easily fixed and good to merge

File: heavenly-visions/Program.cs
Comment (squigggggle) [inline heavenly-visions/Program.cs:1]: out of date code

**Review by squigggggle — CHANGES_REQUESTED** [review]
might need a pull before merging

**Pull request merged**